behaviors/arm/src/arm_nodelet.cc

```diff
@@ -507,83 +536,78 @@ class ArmNodelet : public ff_util::FreeFlyerNodelet {
     dictionary_[name] = GRIPPER;
 
     // Timer for checking goal is reached
-    timer_watchdog_ = nh->createTimer(cfg_.Get<double>("timeout_watchdog"),
-        &ArmNodelet::WatchdogCallback, this, true, false);
+    timer_watchdog_.createTimer(cfg_.Get<double>("timeout_watchdog"),
+        std::bind(&ArmComponent::WatchdogCallback, this), nh, true, false);
 
     // Timer for checking goal is reached
-    timer_goal_ = nh->createTimer(cfg_.Get<double>("timeout_goal"),
-        &ArmNodelet::TimeoutCallback, this, true, false);
+    timer_goal_.createTimer(cfg_.Get<double>("timeout_goal"),
+        std::bind(&ArmComponent::TimeoutCallback, this), nh, true, false);
 
     // Publishers for arm and joint state
-    sub_joint_states_ = nh->subscribe(TOPIC_JOINT_STATES, 1,
-      &ArmNodelet::JointStateCallback, this);
-    pub_joint_goals_ = nh->advertise<sensor_msgs::JointState>(
-      TOPIC_JOINT_GOALS, 1);
+    sub_joint_states_ = FF_CREATE_SUBSCRIBER(nh, sensor_msgs::JointState, TOPIC_JOINT_STATES, 1,
+      std::bind(&ArmComponent::JointStateCallback, this, std::placeholders::_1));
+    // TODO(ana): This had a latch=true argument, but TOPIC_JOINT_GOALS is NOT in the LatchedTopic list
```

Comment:
According to the analysis in https://github.com/nasa/astrobee/issues/628, this does not need to be latched